packages/angular_devkit/build_angular/src/protractor/index.ts

```diff
@@ -105,10 +105,19 @@ async function execute(
     const serverOptions = await context.getTargetOptions(target);
 
     const overrides: Record<string, string | number | boolean> = { watch: false };
-    if (options.host !== undefined) { overrides.host = options.host; }
-    if (options.port !== undefined) { overrides.port = options.port; }
-    server = await context.scheduleTarget(target, overrides);
+    if (options.host && !serverOptions.host) {
```

Comment:
Shouldn't the host option for this builder take precedence like the port option below?